plugins/woo-ai/src/settings.scss

```diff
@@ -1,4 +1,5 @@
 .woo-ai-settings-tone-option {
```

Comment:
Let's move the `settings.scss` and `settings.ts` files into a separate `settings` folder under `src`.